src/Functions/seriesDecomposeSTL.cpp

```diff
@@ -128,6 +128,10 @@ class FunctionSeriesDecomposeSTL : public IFunction
                 res_data.insert(residue.begin(), residue.end());
                 res_col_offsets_data.push_back(res_data.size());
 
+                // Create Baseline = seasonal + trend 
```

Comment:
I think the docs (time-series-functions.md) that the "baseline" really means "seasonal + trend".

Reply:
By the way, does that not add some redundancy, or in other words why can't we have the user do the baseline computation on SQL by her/himself? Perhaps not everyone wants a baseline.

Reply:
Baseline is an important component for time-series analysis representing the predicted value of the series. Its true that it can we easily computed by the user. However, I am explicitly returning it for two reasons:
1) Make the signature/return values of seriesDecomposeSTL() method similar to those of KQL seriesDecompose(https://learn.microsoft.com/en-us/azure/data-explorer/kusto/query/series-decompose-function#returns).
2) The next method in my pipeline `series_decompose_anomalies` would also make use of baseline component to be returned by seriesDecomposeSTL().

Reply:
Okay then.

Please change `std::plus<Float32>()` to `std::plus<>()`, and we are good to go.

(technically, this is a backwards-incompatible change (a function signature changed), but let's not make too much buzz around that since `seriesDecomposeSTL()` is rather specific and rarely used)

Reply:
thank you!